device/mellanox/x86_64-mlnx_msn4600-r0/ACS-MSN4600/port_config.ini

```diff
@@ -0,0 +1,65 @@
+# name          lanes               alias     index
+Ethernet0       0,1,2,3             etp1      1
+Ethernet8       8,9,10,11           etp2      2
```

Comment:
why is this a difference of 8.. its 4 lanes.. @liat-grozovik , @DavidZagury, could you please correct this?